lms/static/js/student_account/views/LoginView.js

```diff
@@ -204,26 +211,21 @@
                 ];
                 this.clearPasswordResetSuccess();
 
-            /* If we've gotten a 403 error, it means that we've successfully
-             * authenticated with a third-party provider, but we haven't
-             * linked the account to an EdX account.  In this case,
-             * we need to prompt the user to enter a little more information
-             * to complete the registration process.
+            /* If the user successfully authenticated with a third-party provider, but they haven't
+             * linked the account to an Open edX account, instruct the user on how to link the accounts.
              */
-                if (error.status === 403 &&
-                 error.responseText === 'third-party-auth' &&
-                 this.currentProvider) {
+                if (errorCode === 'third-party-auth-with-no-linked-account' && this.currentProvider) {
```

Comment:
Question: Since I haven't been following your recent PRs, I'm unaware of this, and I'm sure you covered it in another PR.  So, has `third-party-auth` changed to `third-party-auth-with-no-linked-account`?

Reply:
I did it when I introduced JSON for this error in this recent PR: https://github.com/edx/edx-platform/pull/22452/files

The shim currently is blowing that JSON away and using the earlier `third-party-auth`, but that seemed too vague to me for the error code.